### ISSUE TITLE
`quarto preview` sometimes gets stuck on a reload loop

### ISSUE THREAD

**Issue by cscheid** [quarto-dev/quarto-cli#10382]
In the following instance, the culprit appears to be a `ref: .` entry in the navbar spec.

**Comment by cderv** [comment]
Just curious : 
````yaml
          - text: English
            href: .
````

What does this means exactly ? I would think `href` link for root of the website, but in that case wouldn't it be 
````yaml
          - text: English
            href: /
````

I did not look closely into the code base, but `href` is usually a document or external link. To add a link to root of the website, I would use `/`. Probably equivalent and `.` should work, but maybe catching `.` and replacing by `/` could be a safery measure here... 🤔 

Anyhow, just wanted to share this thoughts of `href: /`

**Comment by mcanouil** [comment]
Indeed, the `href: /` "trick" solved the issue (see discussion on Slack).

My guess is that `href: .` might be resolved on each pages as it means "relative".

**Comment by cscheid** [comment]
Be that as it may, I think there should be nothing in Quarto that causes `quarto preview` to be stuck in a reload loop.

**Comment by max-de-rooij** [comment]
In my project, `quarto preview` remains in a reload loop because it's constantly creating and deleting a folder with a file called `clipboard.min.js`. Globally setting `watch-inputs: false` does not change anything.

**Comment by max-de-rooij** [comment]
@mcanouil Sorry, I've done some investigation and apparently creating a button with a link as `/` causes quarto to reload indefinitely:
```
[Click Me](/){.btn .btn-success}
```

**Comment by mcanouil** [comment]
I can't reproduce.

```sh
quarto preview index.qmd
```

````qmd
---
title: "Quarto Playground"
format: html
---

This is a playground for Quarto.

{{< lipsum 1 >}}

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

[Click Me](/){.btn .btn-success}
````

**Comment by max-de-rooij** [comment]
This is the `yml` file:

```
project:
  type: website

website:
  title: "example"
  navbar:
    background: "#C81919"
    align: center
    title: false
    left:
      - href: index.qmd

format:
  html:
    theme: cosmo
    toc: true
```

**Comment by max-de-rooij** [comment]
https://github.com/user-attachments/assets/24a089f6-1abd-495e-bb0b-b50a8f742d7e

**Comment by mcanouil** [comment]
I still can't reproduce on the development version at least.
